addons/l10n_dk/data/account_financial_report.xml

```diff
@@ -0,0 +1,619 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+        <!-- 
+            (c) 2018 Odoo House ApS
+            Financial Reports
+            display_detail:
+            no_detail
+            detail_flat
+            detail_with_hierarchy
+        -->
+        
+    <!--Delete all old entries-->
+    <delete model="account.financial.report" search="[('type','=','account_report')]"/>
+    <delete model="account.financial.report" search="[('id','>',0)]"/>
+
+    <record id="account_financial_report_resultat" model="account.financial.report">
```

Comment:
All the financial reports must be added as dynamic reports in a dedicated enterprise module called ´l10n_dk_reports´.